selectors/views/browser.ts

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {Constants} from 'utils/constants';
+
+import {GlobalState} from 'types/store';
+import {ViewsState} from 'types/store/views';
+
+export function getBrowserWindowSize(state: GlobalState): ViewsState['browser']['windowSize'] {
+    return state.views.browser.windowSize;
+}
+
+export function getIsDesktopView(state: GlobalState): boolean {
```

Comment:
This might be nitpicking, but I tend to leave off the `get` for selectors that get boolean values. That's because I'm used to the Java convention of boolean getters being named like `isEnabled` instead of `getEnabled`

Reply:
I'm 0/5 on naming conventions here, just trying to follow existing approach ... though it doesn't appear that there is a single, consistent convention.

Interestingly, [Redux recommends prefexing selectors with `select`](https://redux.js.org/style-guide/style-guide#name-selector-functions-as-selectthing).